src/pybind/mgr/nfs/export.py

```diff
@@ -1,5 +1,8 @@
 import errno
 import json
+import stat
+
+import cephfs
```

Comment:
try placing `import cephfs` as the very first statement in this file and see if that resolves the error

Reply:
tried it, didnt help

Reply:
or probably place it right after all system imports but before all local imports
i.e. right after `from os.path import normpath`

Reply:
tried it, but no success